src/javascript/detectors/cross_site_scripting/cross_site_scripting.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// {fact rule=cross-site-scripting@v1.0 defects=1}
+function crossSiteScriptingNoncompliant() {
+    let url = window.location.search.slice(1)
+    // Noncompliant: unsafe jQuery ajax request.
+    $.ajax({url: url, data: "Hello"})
+}
+// {/fact}
+
+
+// {fact rule=cross-site-scripting@v1.0 defects=0}
+const ESAPI = require('node-esapi')
+
+function crossSiteScriptingCompliant() {
+    let url = $(location).attr('hash').slice(1)
```

Comment:
For consistency, can we use the same method of extracting the URL between the two examples? It looks like one is from the window object search params and the other is from hash params using jQuery.